qa/workunits/ceph-helpers.sh

```diff
@@ -117,12 +117,23 @@ function test_setup() {
 function teardown() {
     local dir=$1
     kill_daemons $dir KILL
-    if [ $(stat -f -c '%T' .) == "btrfs" ]; then
+    if [ `uname` != FreeBSD ] \
+        && [ $(stat -f -c '%T' .) == "btrfs" ]; then
         __teardown_btrfs $dir
     fi
     rm -fr $dir
 }
 
+function teardown_error() {
```

Comment:
@dachary 
At the moment it is not used in this change, but I have further changes to keep post-mortum in developer-mode. SO this would beter go with that change as well.